gatsby-node.js

```diff
@@ -283,9 +283,10 @@ class Plugin {
 			Object.assign(headers, (await this.options.headers()) || {});
 		}
 
-		if (this.directus.auth.token) {
+		if (await this.directus.auth.token) {
+			const token = await this.directus.auth.token;
```

Comment:
Lets flip these so we only have to do the async operation once:

```
const token = await this.directus.auth.token;

if (token) {
  ...
}
```

Reply:
done